.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,45 @@
+# Contributing notes
+
+## Pre-requirements: 
+git, make, curl, go, gcc, nginx, mkcert
```

Comment:
Why we need gcc?

Reply:
I started setting up from pmm-admin and it required gcc, so I copy-pasted it to all repos